contracts/burner/src/msg.rs

```diff
@@ -0,0 +1,13 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use cosmwasm_std::HumanAddr;
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct MigrateMsg {
+    pub payout: HumanAddr,
+}
+
+/// EmptyMsg is a placeholder where we don't take any input
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct EmptyMsg {}
```

Comment:
Can we keep the typical structure with `InitMsg` (struct) and `HandleMsg` (empty enum)? I think it is more important to allow people to get orientation easily because all contracts use the same types than having this deduplication.